src/styles/MainTheme.ts

```diff
@@ -28,6 +28,12 @@ export const BaseH1 = styled.h1`
 	font-size: 3em;
 `;
 
+export const CustomH1 = styled.h1`
```

Comment:
zgaduje ze na h3 się obraziłeś, ale jesli wyglada ok wizualnie to mi to lotto.

Reply:
Może i tak, ale możesz też zrobić styled.h3. Nevertheless, jeśli to wg ciebie działa to mi to lotto